applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -376,15 +400,15 @@ def test_compare_TL_UL_2D_triangle(self):
                 ul_dx = ul_mp.Nodes[i].GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_X)
                 ul_dy = ul_mp.Nodes[i].GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_Y)
 
-                if (ul_dx > 0.0):
-                    self.assertLess((tl_dx - ul_dx) / ul_dx, 1.0e-3)
-                if (ul_dy > 0.0):
-                    self.assertLess((tl_dy - ul_dy) / ul_dy, 1.0e-3)
+                if(abs(tl_dx) > 1e-20):
```

Comment:
the relative difference between TL and UL should be zero for an elastic material up to machine precision...

@loumalouomega

Reply:
Ok